lib/tictactoe.rb

```diff
@@ -20,6 +20,16 @@ def initialize (game_io, player1, player2)
   end
 
   def play_game
+    begin
     @game_loop.game_loop
+    @board.new_game
+    end while (play_again?)
+  end
+
+  def play_again?
+    @io.puts_message("Play again (Y/N) : ")
+    input = @io.get_input
+    return true if (input == "y" || input == "Y")
+    false
```

Comment:
You have this `return true if ...  \n false` pattern in a few places. Since equality checks and logical operators both return booleans in Ruby, you can simplify this to just `input == "y" || input == "Y"`. There are a few other places where you could take advantage of OR's behavior to "cascade" through possible return values.